### PR TITLE
Use conditional whiskers

### PR DESCRIPTION
**Pull request by chriseth**
Depends on https://github.com/ethereum/solidity/pull/6688

### CODE REVIEW

**Comment by stackenbotten** [comment]
There was an error when running `test_check_style` for commit `dcbfb20ad1a0b7913b3bee8e8990d82f9f51eec5`:
```
Coding style error:
 libsolidity/codegen/YulUtilFunctions.cpp:282: sum := add(x, y)
 libsolidity/codegen/YulUtilFunctions.cpp:283: if lt(sum, x) { revert(0, 0) }

```
Please check that your changes are working as intended.

**Comment by leonardoalt** [comment]
Please rebase

**Comment by stackenbotten** [comment]
There was an error when running `test_check_style` for commit `e80fddee8c50bb58f1e2bca2959692a30dd054e4`:
```
Coding style error:
 libsolidity/codegen/YulUtilFunctions.cpp:336: sum := add(x, y)
 libsolidity/codegen/YulUtilFunctions.cpp:337: if lt(sum, x) { revert(0, 0) }

```
Please check that your changes are working as intended.

File: test/libsolidity/gasTests/abiv2.sol [hunk 1]
```diff
@@ -13,7 +13,7 @@ contract C {
     function f8(uint[32] memory, string[] memory, uint32, address) public returns (uint[] memory, uint16[] memory) {}
 }
 // ----
-// creation: 1160 + 1119000 = 1120160
```
Comment (chriseth) [inline test/libsolidity/gasTests/abiv2.sol]: This is due to an additional assignment in the Yul code. The optimizer should take care of that later.

**Comment by leonardoalt** [comment]
Apparently the test needs another update

**Review by leonardoalt — APPROVED** [review]

**Pull request merged**